docs/references/javascript/clerk/clerk.mdx

```diff
@@ -7,7 +7,7 @@ description: This is the main entrypoint class for the `@clerk/clerk-js` package
 
 This is the main entrypoint class for the `@clerk/clerk-js` package. It contains a number of methods and properties for interacting with the Clerk API.
 
-## Properties
+## `Clerk` properties

```

Comment:
Any specific reason for this? For a sec I thought this would help with search, but actually if I search "clerk properties" in prod I get this:
![image](https://github.com/clerk/clerk-docs/assets/5156915/814823a6-25d1-4128-980e-4f5e63ddf601)

So I think adding `Clerk` here is redundant

Reply:
Okay I'll remove it!